src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -31,14 +33,17 @@ const ENDPOINT_MAP = {
   userRepostsByHandle: (handle: OpaqueID) => `/users/handle/${handle}/reposts`,
   getPlaylist: (playlistId: OpaqueID) => `/playlists/${playlistId}`,
   topGenreUsers: '/users/genre/top',
-  track: (trackId: OpaqueID) => `/tracks/${trackId}`
+  track: (trackId: OpaqueID) => `/tracks/${trackId}`,
```

Comment:
should we call the getTrack, getStems, etc? to be consistent? or maybe rename the others

Reply:
will add verb